### PR TITLE
[121] Update libraries batch-2

### PR DESCRIPTION
**Pull request by stanislaw-zakrzewski**
Partly solves: #121

### CODE REVIEW

File: verification/curator-service/ui/src/components/landing-page/LandingPage.test.tsx [hunk 1]
```diff
@@ -96,7 +97,7 @@ describe('<LandingPage />', () => {
 });
 
 describe('<SignInForm />', () => {
-    it('renders and submits form', async () => {
+    it.skip('renders and submits form', async () => {
```
Comment (abhidg) [inline verification/curator-service/ui/src/components/landing-page/LandingPage.test.tsx:100]: Are these temporary skips? Will be good to create an issue to keep track if it needs to be un-skipped

File: verification/curator-service/Dockerfile [hunk 2]
```diff
@@ -42,12 +42,12 @@ ENV REACT_APP_PUBLIC_MAPBOX_TOKEN "pk.eyJ1IjoiaGVhbHRobWFwIiwiYSI6ImNrYzNjczdmcz
...
 ENV REACT_APP_RECAPTCHA_SITE_KEY "6LdhjvwgAAAAALrxavR_zR58kxxap07D4ba8X-jE"
-ENV REACT_APP_DISEASE_NAME "Mpox"
-ENV REACT_APP_MAP_LINK_DEV "http://dev-map.marburg.global.health/"
-ENV REACT_APP_MAP_LINK_LOCAL "http://dev-map.marburg.global.health/"
-ENV REACT_APP_MAP_LINK_LOCAL_E2E "http://dev-map.marburg.global.health/"
-ENV REACT_APP_MAP_LINK_PROD "http://dev-map.marburg.global.health/"
-ENV REACT_APP_MAP_LINK_QA "http://map.marburg.global.health/"
+ENV VITE_APP_DISEASE_NAME "Mpox"
+ENV VITE_APP_MAP_LINK_DEV "http://dev-map.marburg.global.health/"
+ENV VITE_APP_MAP_LINK_LOCAL "http://dev-map.marburg.global.health/"
```
Comment (abhidg) [inline verification/curator-service/Dockerfile:47]: Can these be left blank or made consistent? Right now disease is Mpox but URLs are marburg

File: verification/curator-service/ui/src/components/landing-page/LandingPage.test.tsx (hunk 1 above)
Comment (stanislaw-zakrzewski) [reply verification/curator-service/ui/src/components/landing-page/LandingPage.test.tsx:100]: Yes, I indended to create an issue fot "unskipping" the tests, but I must have forgotten. Here it is:
https://github.com/globaldothealth/turnkey-curator-portal/issues/129

File: verification/curator-service/Dockerfile (hunk 2 above)
Comment (stanislaw-zakrzewski) [reply verification/curator-service/Dockerfile:47]: This is an issue that I and Jim found during deploying the app. In order to deploy build is run on the CuratorUI and for build env variables are set. This problem has been already solved partially for the other variables like disease name, which is queried from backend. Same solution can be employed also for map related variables. I can create an issue for that.

**Pull request merged**